azure/durable_functions/models/DurableOrchestrationContext.py

```diff
@@ -158,19 +161,48 @@ def call_activity(self, name: str, input_: Optional[Any] = None) -> TaskBase:
         Task
             A Durable Task that completes when the called activity function completes or fails.
         """
+        if(isinstance(name, Callable) and not isinstance(name, FunctionBuilder)):
```

Comment:
I think you can remove the if-condition parenthesis. They shouldn't be needed AFAIK.
```suggestion
        if isinstance(name, Callable) and not isinstance(name, FunctionBuilder):
```

Here and in the places below.